### ISSUE TITLE
Falling through world via blink: Shattrath Scryer Bank

### ISSUE THREAD

**Issue by Liesara** [ccshiro/cc-buglist#1829]
When I tried blinking toward the left Scryer gbank I fell through the world and without getting an option to release spirit immediately ended up at the spirit healer once I died. This seems to be reproducable by flying into the bank about as far as you can (so you need to do about a 90 degree turn toward the gbank) and then blink SLIGHTLY to the left of the gbank before hitting the ground. You end up beneath the floor of the bank and if you're not careful end up falling to your death. It's very iffy though, I tried about 30 or 40 times and only got it right 3 or 4 times.

You can also blink inside the gbank vault, though that doesn't trigger any other problems.

**Comment by ghost** [comment]
I have tried to recreate what you have decribed but without luck. Could you perhaps provide a video clip of it?

**Comment by Liesara** [comment]
Yeah I thought I'd have to. Guess I'll see to it to set something up. Will try to get it done tomorrow. As i said though, it's very hard to pull off, seems it needs a really specific angle.

**Comment by Liesara** [comment]
https://youtu.be/8sA2XzUao2M

Sorry for the cancerous quality but I guess you can see what's happening. Got lucky and got it right on the second try.

**Comment by Liesara** [comment]
This also happened to me when blinking from the gbank in the direction of the counter. Can't reproduce it however.
